src/App.js

```diff
@@ -14,7 +14,8 @@ import ErrorPage from './views/ErrorPage';
 import { useAppSettings } from './store/store.js';
 import { PrintContextProvider } from './store/printContext';
 import { useAppInitializer } from './hooks';
-import Layout from './views/Layout';
+//import Layout from './views/Layout';
```

Comment:
If `Layout` is no longer used, can you remove from here, and probably the component itself since this is the only place it's used in the codebase.